cou/cli.py

```diff
@@ -291,4 +295,6 @@ def entrypoint() -> None:
     finally:
         if args.command == "upgrade":
             loop.run_until_complete(run_post_upgrade_sanity_check(args))
+        if log_file is not None:
+            progress_indicator.stop_and_persist(text=f"Full execution log: '{log_file}'")
```

Comment:
Missed this before - this should also be a plain `print`, not the progress_indicator. Maybe a few things were lost in the rebase? :sweat_smile:

Reply:
Updated